.circleci/config.yml

```diff
@@ -3,28 +3,78 @@ jobs:
   lint:
     docker:
       - image: circleci/golang:1.12-node
-
     working_directory: /go/src/github.com/mattermost/mattermost-plugin-jira
     steps:
       - checkout
-
       - run: curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
       - run: make check-style
 
   test:
     docker:
       - image: circleci/golang:1.12-node
-
     working_directory: /go/src/github.com/mattermost/mattermost-plugin-jira
     steps:
       - checkout
-
       - run: curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
       - run: make test
 
+  build:
+    docker:
+      - image: circleci/golang:1.12-node
+    working_directory: /go/src/github.com/mattermost/mattermost-plugin-jira
```

Comment:
See my comment in https://github.com/mattermost/mattermost-plugin-github/pull/60#discussion_r264269279.

Otherwise LGTM.

Reply:
done @hanzei